### ISSUE TITLE
"There is incomplete content above" notice for screen readers is present regardless of Trickle button hidden/shown

### ISSUE THREAD

**Issue by sebastian-onlea** [adaptlearning/adapt-contrib-trickle#141]
It looks as though `{{{a11y_aria_label _globals._extensions._trickle.incompleteContent}}}` in templates/trickle-button.hbs was left outside the conditionalized rendering in the last update, causing the notice for screen reader users to be included even when the Trickle button is not visible or accessible.

**Comment by oliverfoster** [comment]
I can't replicate this. 
[this line](https://github.com/adaptlearning/adapt-contrib-trickle/blob/master/js/TrickleButtonView.js#L129) seems to appropriately add aria-hidden appropriately which will remove the div from the screen reader's view.

![image](https://user-images.githubusercontent.com/7974663/157228609-37bdc11b-9b91-4f1f-bba0-6b9002c7fc1a.png)

Could you give me a better idea of how to replicate this please?

**Comment by guywillis** [comment]
https://github.com/adaptlearning/adapt-contrib-trickle/blob/master/js/TrickleButtonModel.js#L156

Change to:
`_isButtonDisabled: !this.isStepUnlocked()`
